sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -487,6 +487,11 @@ export class MessageSession extends LinkEntity {
           "'session lock renewal' task.",
         this._context.namespace.connectionId
       );
+
+      if (isSessionExpired === undefined || !isSessionExpired) {
```

Comment:
this can be simplified to `if (!isSessionExpired)` because `!undefined` is `true`

Reply:
I thought so too, but the check was specifically failing for when value was passed in as `undefined`.

Reply:
With latest changes, this got simplified as it no longer takes undefined as a possible value.